### PR TITLE
Upgrade from raven-js to sentry/browser 

### LINKED ISSUE #2412: Help upgrading from raven-js
We are using a deprecated https://www.npmjs.com/package/raven-js package in [packages/jaeger-ui/src/utils/tracking/](./packages/jaeger-ui/src/utils/tracking/). Maybe we can use `@sentry/browser` for that, or maybe we can just implement the needed functionality internally since we use it only to capture some basic error information to submit to the tracking backend.

### PR DESCRIPTION
**Pull request by avinpy-255**
## Which problem is this PR solving?
- Resolves #2412
- Replaced and closes #2417 

## Description of the changes
This PR is a continuation of the migration from `raven-js` to `@sentry/browser`.
- The final step involves testing the implementation with a real Google Analytics account.
- The google tags are displayed correctly in [Tag Hound](https://chromewebstore.google.com/detail/taghound-analyticsgtmpixe/canpneabbfipaelecfibpmmjbdkiaolf?hl=en) chrome extension.
- Here is the screenshot of logging tag managers details and event data.
Events from TagHound

## How was this change tested?
- Placed the GA Measurement ID in `default-config.tsx`.
- Varified navigation events using TagHound. Screenshots of the events been tracked are attached
- **Note**: Ensure GTM tracking is enabled in TagHound, otherwise the Google tags will not be tracked.
-    This test works in the local environment, but I am unable to see data logs in Google Analytics. This may be because the GA dashboard doesn’t collect data from localhost.

## Checklist
- [x] I have read https://github.com/jaegertracing/jaeger/blob/master/CONTRIBUTING_GUIDELINES.md
- [x] I have signed all commits
- [x] I have added unit tests for the new functionality
- [x] I have run lint and test steps successfully
  - for `jaeger`: `make lint test`
  - for `jaeger-ui`: `npm test`

### CODE REVIEW

**Comment by yurishkuro** [comment]
Please add testing instruction to packages/jaeger-ui/src/utils/tracking/README.md instead of only having in the PR.

File: packages/jaeger-ui/src/utils/tracking/README.md [hunk 1]
```diff
@@ -222,3 +222,19 @@ You get a lot for free when using Raven.js:
 - Some global handlers are added
 
 Implementing the above from scratch would require substantial effort. Meanwhile, Raven.js is well tested.
+
+### Steps to Verify the gaID Integration
+
+1.  **Setup**
+
+Ensure you have the repository cloned and all dependencies installed. Run the following command in the terminal:
+`yarn install`
```
Comment (yurishkuro) [inline packages/jaeger-ui/src/utils/tracking/README.md:231]: we don't use yarn anymore
```suggestion
`npm install`
```

File: packages/jaeger-ui/src/utils/tracking/README.md [hunk 2]
```diff
@@ -222,3 +222,19 @@ You get a lot for free when using Raven.js:
 - Some global handlers are added
 
 Implementing the above from scratch would require substantial effort. Meanwhile, Raven.js is well tested.
+
+### Steps to Verify the gaID Integration
+
+1.  **Setup**
```
Comment (yurishkuro) [inline packages/jaeger-ui/src/utils/tracking/README.md:228]: this is not a valid formatting for markdown, your bullets are not rendering correctly

Comment (yurishkuro) [reply packages/jaeger-ui/src/utils/tracking/README.md:228]: I suggest using single line, except for (3) which seems fine

File: packages/jaeger-ui/src/utils/tracking/README.md [hunk 3]
```diff
@@ -222,3 +222,19 @@ You get a lot for free when using Raven.js:
...
+
+Ensure you have the repository cloned and all dependencies installed. Run the following command in the terminal:
+`yarn install`
+2.  **Start the Local Development Server**
+
+`yarn start`
+   This will open the application in your default browser.
+3.  **Steps to Put gaID**
+- Place the GA Measurement ID in `default-config.tsx`.
```
Comment (yurishkuro) [inline packages/jaeger-ui/src/utils/tracking/README.md:237]: sub-list must be indented in markdown
```suggestion
  - Place the GA Measurement ID in `default-config.tsx`.
```

File: packages/jaeger-ui/src/utils/tracking/README.md (hunk 2 above)
Comment (yurishkuro) [reply packages/jaeger-ui/src/utils/tracking/README.md:228]:
```suggestion
1. Setup: ...
```

**Review by yurishkuro — APPROVED** [review]

**Comment by yurishkuro** [comment]
Thanks @BenzeneAlcohol and @avinpy-255

**Comment by avinpy-255** [comment]
You're welcome, and thank you for your guidance and support throughout! I really learned a lot during this process and look forward to contributing more in the future!

**Pull request merged**